tools/run_tests/generated/OWNERS

```diff
@@ -0,0 +1,5 @@
+set noparent
+
+# Files in this directory are generated. No owners needed.
+
+*
```

Comment:
tools/run_tests/generated    shouldn't require owners approval, and I was hoping this would do it, but I'm actually unsure if github will accept that. Also unsure what to be the best way to test this (without actually merging).